script.js

```diff
@@ -1,4 +1,86 @@
+outputValue = `Player ${currentPlayer}, your chosen value is ${playerScore}! `;
```

Comment:
This variable seems out of place. Also, it does not have the `var` keyword.
I don't think you really need this variable to be global